src/libcore/num/dec2flt/algorithm.rs

```diff
@@ -32,19 +32,101 @@ fn power_of_ten(e: i16) -> Fp {
     Fp { f: sig, e: exp }
 }
 
+// Most architectures floating point operations with explicit bit size, therefore the precision of
+// the computation is determined on a per-operation basis.
+#[cfg(any(not(target_arch="x86"), target_feature="sse2"))]
+mod fpu_precision {
+    pub fn set_precision<T>() { }
+}
+
+// On x86, the x87 FPU is used for float operations if the SSE[2] extensions are not available.
```

Comment:
`SSE[2]` &mdash; I initially parsed the [2] as a citation, perhaps "SSE/SSE2" is better.